### DISCUSSION TITLE
Arc and Arc-Dark themes

### DISCUSSION THREAD

**Discussion by AzumaHazuki** [lxqt/lxqt#2337] [Show and tell]
Fresh out of the oven tonight :) Overall I like the looks of these so far. Includes also 8-color palettes for each.

Edit: Final (hopefully) screenshots here: 

![Arc](https://user-images.githubusercontent.com/67122280/202884505-4993f110-359f-45c9-bed9-1fafe545351a.jpg)
![Arc-Dark](https://user-images.githubusercontent.com/67122280/202884508-9bee3da2-8e3b-4497-9913-b8d19735aaa6.jpg)

**Comment by stefonarch** [comment]
We accept PR's ;)

**Reply by AzumaHazuki** [reply]
Edit: Repository <a href=https://github.com/AzumaHazuki/lxqt-themes-arc>here</a>. It may be more useful to have Adwaita themes rather than Arc themes in the default set; given the rate at which I've been creating themes and palettes, there's danger of clogging the theme selector up ><
